crates/alpen-ee/common/src/types/prover.rs

```diff
@@ -0,0 +1,9 @@
+use strata_acct_types::Hash;
+
+/// Unique identifier to a persisted proof.
+pub type ProofId = Hash;
+
+/// TODO: proper proof type
+#[expect(dead_code, reason = "wip")]
+#[derive(Debug)]
+pub struct Proof(Vec<u8>);
```

Comment:
Do we need this to grow dynamically?

```suggestion
pub struct Proof(&[u8]);
```